exercises/rational-numbers/test/test_rational_numbers.c

```diff
@@ -0,0 +1,439 @@
+#include "vendor/unity.h"
+#include "../src/rational_numbers.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void check_rational_numbers_equal(rational_t expected, rational_t actual)
+{
+   TEST_ASSERT_EQUAL_INT16(expected.numerator, actual.numerator);
+   TEST_ASSERT_EQUAL_INT16(expected.denominator, actual.denominator);
```

Comment:
Is there a way to have an error message to make it easier to know which of the numerator/denominator is wrong? I think there's a version of the assert with a string as a third argument.

Reply:
Yes - I thought about adding it earlier but forgot! 🤦‍♂